Frontend/ui-driver/src/Screens/RegistrationScreen/View.purs

```diff
@@ -179,6 +181,7 @@ view push state =
                                       _ -> VISIBLE
                                   , background case getStatus item.stage state of
                                       ST.COMPLETED -> Color.green900
+                                      ST.MANUAL_VERIFICATION_REQUIRED -> Color.green900
```

Comment:
For Manual Verification Required, if `default_manual_verification_verified` is kept as false in db, then what should be the case here because complete registration won't be coming but the color would be showing green.